include/AutomationNode.h

```diff
@@ -27,12 +27,15 @@
 #define AUTOMATION_NODE_H
 
 
+class AutomationPattern;
+
+
 class AutomationNode
 {
 public:
-	AutomationNode();
-	AutomationNode( float value );
-	AutomationNode( float inValue, float outValue );
+	AutomationNode(); // Dummy constructor for the QMap
+	AutomationNode( AutomationPattern * pat, float value, int key );
+	AutomationNode( AutomationPattern * pat, float inValue, float outValue, int key );
```

Comment:
I forget what we decided for pointers, but I stuck it on the type for now.
```suggestion
	AutomationNode(AutomationPattern* pat, float value, int key);
	AutomationNode(AutomationPattern* pat, float inValue, float outValue, int key);
```